### DISCUSSION TITLE
Switch to net8

### DISCUSSION THREAD

**Discussion by Gauvino** [jellyfin/jellyfin-meta#57] [Ideas]
Hello, does jellyfin gonna keep net7 for the 10.9 and get the net8 for 10.10 or your just waiting until net8 released and switch to it right after?

**Comment by Bond-009** [comment]
10.9 will be .NET8, we usually wait till the full release to switch

**Reply by Gauvino** [reply]
Thanks you for the fast reply and cool to see that the project is that much updated!
You guys are doing such a great work, and I really thank for it jellyfin is my favorite project at all.
